consumer-java/aws-kinesis-video-producer-sdk-canary-consumer/run-consumer.sh

```diff
@@ -1,8 +1,7 @@
 mvn package
```

Comment:
I'd highly recommend to use Makefile instead so we can even add this to our CI. 

To add this to the CI, you just need to create a Makefile that has "test" as the target, and add the directory path to the target in .travis.yaml below:

https://github.com/aws-samples/amazon-kinesis-video-streams-demos/blob/e246e81370751088cb82b32be6e5176bb413d0f7/.travis.yml#L17

It's definitely not the best test, but it can add some sanity check at least.

Reply:
Webrtc Canary's Makefile, https://github.com/aws-samples/amazon-kinesis-video-streams-demos/blob/master/webrtc-c/canary/Makefile

Reply:
Let me handle this in a seperate PR. As of now, run-consumer.sh is not being used in our Jenkins pipeline